### ISSUE TITLE
Limit of analysis to be copied

### ISSUE THREAD

**Issue by Felipe-UnB** [CIRDLES/Topsoil#241]
Heys guys, how are you? So, I was trying to copy a lot of results from excel to TOPSOIL (exactly 330 analysis). However, when I try to paste from the clipboard, there is a limit of 239 analysis that can be copied. Is this a real limit or it might be some kind of bug? I am using Windows 10 Home and Microsoft Office Professional Plus 2010.

**Comment by johnzeringue** [comment]
We've yet to see this sort of thing before. If it's a problem with Topsoil, it's certainly not intentional. This could also be an issue with Windows, Excel, or your data. Are you able to send me the file you're having trouble with? If I'm able to reproduce the issue, I can start working on a fix or a workaround.

**Comment by Felipe-UnB** [comment]
Hey John, I will prepare the file ASAP!

**Comment by Felipe-UnB** [comment]
[Resultados 91500 - TOPSOIL issue.xlsx](https://github.com/CIRDLES/Topsoil/files/41621/Resultados.91500.-.TOPSOIL.issue.xlsx)

**Comment by Felipe-UnB** [comment]
I don´t believe the problem is actually with the data because I was able to plot it using Isoplot. But let´s see what you find out, maybe Isoplot is handling some kind of error in a way that It´s not obvious to me.

**Comment by johnzeringue** [comment]
I've found the issue and will have a fix out soon. @Felipe-UnB, thanks for reporting this.

**Comment by Felipe-UnB** [comment]
John, the bug was actually related to the number of the analyses?

**Comment by johnzeringue** [comment]
Not the one I found, but I could reproduce the copy-paste failing. After that, everything worked fine. Are you still having issues?

**Comment by Felipe-UnB** [comment]
The version with the bug correction for this issue is the v1.0.0-alpha.6, correct? If that's right, I am still experiencing the same problem. I will try exporting the data as CSV and see if I am able to plot all of them.

**Comment by johnzeringue** [comment]
I'll try again to reproduce and get back to you.
